proto/ojo/gmp/v1/tx.proto

```diff
@@ -46,8 +46,8 @@ message MsgRelayPrice {
   // destination_chain defines the chain which this will be relayed to.
   string destination_chain = 2;
 
-  // destination_address defines the destination contract address to call.
-  string destination_address = 3;
+  // contract_address defines the destination contract address to call.
```

Comment:
I think we need the destination contract address too (`destination_Address`: ojo contract and `contract_Address`: client contract)